device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/DeviceClient.java

```diff
@@ -376,6 +378,11 @@ private void commonConstructorVerifications() throws UnsupportedOperationExcepti
      */
     public void open() throws IOException
     {
+        if (this.ioTHubConnectionType == IoTHubConnectionType.USE_MULTIPLEXING_CLIENT)
+        {
+            throw new UnsupportedOperationException("Cannot open a multiplexed client through this method. Must use multiplexingClient.registerDeviceClients(deviceClient)");
```

Comment:
nit: this message could be in a const string, similar to `MULTIPLEXING_CLOSE_ERROR_MESSAGE`.